magazineApp/magazineApp/Controllers/WebStoreViewController.swift

```diff
@@ -0,0 +1,149 @@
+//
+//  WebStoreViewController.swift
+//  magazineApp
+//
+//  Created by 2lup on 10.10.2022.
+//
+
+import UIKit
+import WebKit
+
+/// Cтраница интернет магазина
+class WebStoreViewController: UIViewController, WKUIDelegate {
```

Comment:
```suggestion
final class WebStoreViewController: UIViewController {
```
WKUIDelegate в extension